_posts/2017/2017-11-12-use-a-pull-request-template.markdown

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Use a pull request template'
+description: 'It helps you answer what, why & how.'
+layout: article
+category: ruby
+image:
+  base: '2017/use-a-pull-request-template'
+  alt: 'Man pulling rope'
+  credit: 'Stijn Swinnen'
+  source: 'https://unsplash.com/photos/Q8FHN3qSq2w'
+---
+
+Pull Requests are the primary place where you explain your work. The audience for a Pull Request can be your immediate co-workers, other interested parties or even your future self.
+
+It should describe the ‘what’, ‘why’ & high-level ‘how’ of the changes in your branch. It is important the description is detailed and gives the right level of contextual detail.
+
+Github offers the ability to define issue and pull request templates. See [GitHub’s documentation on Pull Request templates](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/).
```

Comment:
Change 'Pull Request' to 'pull request'.

Reply:
Maybe take out the reference to 'issues' here and just stick to GitHub offers the ability to define pull request templates. Find out more in their documentation.

It's minor, but no need to introduce this other concept of GitHub issues.